iam.tf

```diff
@@ -126,6 +126,12 @@ resource "aws_iam_policy_attachment" "network" {
 
 # Attach an additional policy if provided.
 
+resource "aws_iam_role_policy_attachment" "additional" {
```

Comment:
Please move this down to below the other `aws_iam_policy_attachment` and rename it to `additional_arn` as there is already one named `additional`.

Reply:
Move it, but kept the name, as the original "additional" was no longer needed.